src/idpi/operators/crop.py

```diff
@@ -0,0 +1,63 @@
+"""Horizontal cropping operator."""
+
+# Third-party
+import xarray as xr
+
+# Local
+from .. import metadata
+from . import gis
+
+
+def crop(field: xr.DataArray, bounds: tuple[int, ...]) -> xr.DataArray:
+    """Crop the field to the given bounds.
+
+    Only fields defined on regular grids in rotlatlon coordinates,
+    without rotation nor flipped axes are supported.
```

Comment:
I do not understand the remark "without rotation here".

Reply:
The rotated lat lon coordinate system can have an additional rotation around the vertical at the origin point.

Reply:
OK!